src/agent/coverage/examples/src-cov.rs

```diff
@@ -22,7 +22,7 @@ struct Opt {
     #[structopt(min_values = 2)]
     cmd: Vec<String>,
 
-    #[structopt(short, long, long_help = "Timeout in ms", default_value = "5000")]
+    #[structopt(short, long, long_help = "Timeout in ms", default_value = "120000")]
```

Comment:
Relevant task executor constant can be found [here](https://github.com/microsoft/onefuzz/blob/cafba18c8500bce241a7480880ae6f2e9717a31a/src/agent/onefuzz-task/src/tasks/coverage/generic.rs#L39). May want to check for any places where it gets set server-side or in the CLI.

Reply:
(the attr here never used within OneFuzz, it's just used by the example binary for debugging)

Reply:
doh!